uint8 start_vertex
---
uint8 next_vertex
uint8 edge_weight
